lib/matplotlib/quiver.py

```diff
@@ -85,6 +85,13 @@
     height of the axes, respectively, when the the window is resized;
     for 'dots' or 'inches', resizing does not change the arrows.
 
+  *orientation*: ['absolute' | 'relative']
```

Comment:
I'm still on the fence about this.  The "absolute" mode makes the U-V coordinate system independent of the X-Y coordinate system, and this makes logical sense in some cases, such as when they really are completely different things.  For example, quiver could be used to make a bunch of clock faces, to show times at X-Y locations that might have nothing to do with time.  For that case, one would not want the clocks to flip with the  X or Y axes.  So, I think the best course of action might be to add the kwarg as in the present PR and leave the default as the old behavior for 1.4.3 and maybe 1.5, and then switch the default in 1.5 or 1.6 or 2.0.  But if you and others are comfortable with switching the default now, I could go along with that.  It just seems marginal for a bugfix release.